eva_sub_cli/validators/docker_validator.py

```diff
@@ -24,9 +25,8 @@ def __init__(self, mapping_file, submission_dir, project_title, metadata_json=No
                          metadata_json=metadata_json, metadata_xlsx=metadata_xlsx,
                          shallow_validation=shallow_validation, submission_config=submission_config)
         self.docker_path = docker_path
-        self.container_name = container_name
```

Comment:
Maybe just remove the `container_name` param, unless you think it might be useful later...